src/components/Navbar.js

```diff
@@ -1,17 +1,21 @@
 import { Link } from 'react-router-dom';
 import styles from './Navbar.module.css';
+import user from '../images/user.svg';
 
 const Navbar = () => (
-  <nav>
-    <h1 className={styles.title}><Link className={styles.link} to="/">Bookstore</Link></h1>
-    <ul className={styles.linkCont}>
-      <li className={styles.linkEl}>
-        <Link className={styles.link} to="/">Books</Link>
-      </li>
-      <li className={styles.linkEl}>
-        <Link className={styles.link} to="/categories">Categories</Link>
-      </li>
-    </ul>
+  <nav className={styles.navCont}>
+    <div className={styles.linkCont}>
+      <h1><Link className={styles.titleLink} to="/">Bookstore CMS</Link></h1>
+      <ul className={styles.linkCont}>
+        <li className={styles.linkEl}>
+          <Link className={styles.link} to="/">BOOKS</Link>
+        </li>
+        <li className={styles.linkEl}>
+          <Link className={styles.link} to="/categories">CATEGORIES</Link>
+        </li>
+      </ul>
+    </div>
+    <img className={styles.user} alt="user account logo" src={user} />
   </nav>
 );
```

Comment:
- [ ] Hi, good job @Anjacodes however let us try to meet the requirements as well as the design provided. As you can see the menu or navbar div at the original design has the same width as the div or card that has the information added. 

Original design

![header](https://user-images.githubusercontent.com/55635977/150568500-84de5501-be44-41f8-a3d7-308d3ed1e469.png)

Your design

![header2](https://user-images.githubusercontent.com/55635977/150568522-28a8885b-044a-43fd-b8d8-28ec9df774df.png)